cli/kv_command.go

```diff
@@ -787,6 +800,15 @@ func (c *kvCommand) showStatus(store nats.KeyValue) error {
 
 			fmt.Println()
 		}
+
+		if len(nfo.Sources) > 0 {
+			fmt.Println("\n Sources Information:")
+			fmt.Println()
+			for _, source := range nfo.Sources {
+				fmt.Printf("        Source Bucket: %s\n", strings.TrimPrefix(source.Name, "KV_"))
```

Comment:
Think it would need to be like stream info - showing lag and last seen also per source

Reply:
Indeed, done.